src/pages/workspace/WorkSpaceUtils.js

```diff
@@ -0,0 +1,15 @@
+import lodashGet from 'lodash/get';
+import BankAccount from '../../libs/models/BankAccount';
+import Navigation from '../../libs/Navigation/Navigation';
+
+function getShouldShowPage(props, nextPage, defaultPage) {
```

Comment:
Name the should on what it does?

Reply:
Changed. I change with `getShouldShowBankOrNonAccountPage` Please suggest is it okay?